### PR TITLE
Paypal migration to v10

### PR DESCRIPTION
**Pull request by rvalyi**
do not merge yet, work in progress...
Tests passing on v10.

### CODE REVIEW

**Comment by rvalyi** [comment]
ready for merge

File: payment_gateway_paypal/services/payment_service.py [hunk 1]
```diff
@@ -3,9 +3,10 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models
+from odoo.exceptions import Warning as UserError
+from odoo.addons.component.core import Component
+
 import json
-from openerp.exceptions import Warning as UserError
 import logging
 _logger = logging.getLogger(__name__)
```
Comment (sebastienbeau) [inline payment_gateway_paypal/services/payment_service.py:12]: We should migrate and refactor the profile creation, (line 20) method "def create_profile".
We should create a wizard that will give the posibility ro configure the profile

**Comment by lmignon** [comment]
superseded by #34

**Comment by sebastienbeau** [comment]
Merge in #34

**Pull request closed without merging**